extensions/amp-fit-text/1.0/README.md

```diff
@@ -0,0 +1,218 @@
+    document.querySelector('#content-button').onclick = () => fitText.textContent = 'new content';
+  })();
+</script>
+```
+
+[/example]
+
+#### Overflowing content
+
+If the content of the `bento-fit-text` overflows the available space, even with a
+`min-font-size` specified, the overflowing content is cut off and hidden. WebKit and Blink-based browsers show ellipsis for overflowing content.
+
+In the following example, we specified a `min-font-size` of `40`, and added more content inside the `bento-fit-text` element. This causes the content to exceed the size of its fixed block parent, so the text is truncated to fit the container.
+
+[example preview="inline" playground="true" imports="bento-fit-text:1.0"]
+
+```html
+<div style="width: 300px; height: 300px; background: #005AF0; color: #FFF;">
+  <bento-fit-text min-font-size="40">
+    Lorem ipsum dolor sit amet, has nisl nihil convenire et, vim at aeque
+    inermis reprehendunt. Lorem ipsum dolor sit amet, has nisl nihil convenire
+    et, vim at aeque inermis reprehendunt
+  </bento-fit-text>
+</div>
+```
+
+[/example]
+
+#### Layout and style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-fit-text-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  bento-fit-text {
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+</style>
+```
+
+**Container type**
+
+The `bento-fit-text` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children (slides) via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties):
+
+```css
+bento-fit-text {
+  height: 100px;
+  width: 100%;
+}
+```
+
+#### Accessibility considerations of overflowing content
+
+While overflowing content is _visually_ truncated to fit the container, note that it's still present in the document. Do not rely on the overflow behaviour to simply "stuff" large amounts of content in your pages - while visually it may look appropriate, it may lead to the page becoming overly verbose to users of assistive technologies (such as screen readers), as for these users all the truncated content will still be read/announced in full.
+
+#### Attributes
+
+##### Media Queries
+
+The attributes for `<bento-fit-text>` can be configured to use different
+options based on a [media query](./../../../docs/spec/amp-html-responsive-attributes.md).
+
+##### `min-font-size`
+
+Specifies the minimum font size as an integer that the `bento-fit-text` can use.
```

Comment:
I'm assuming this is in pixels, but is it worth calling out the units here?

Reply:
Good idea, updated in all locations.